From f5d37650e7077d214ec5895b0175918863172086 Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Tue, 13 Sep 2022 09:36:38 +0200
Subject: [PATCH 34/N] fixup! Revert "Default to `disable_pcon`"

This reverts commit 8e89fffcfb0884da1398dd55f0d0cc57294549ec.

We want to try enabling it by default again, see #98
---
 winsup/cygwin/globals.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/winsup/cygwin/globals.cc b/winsup/cygwin/globals.cc
index 6c15138..6a65dd6 100644
--- a/winsup/cygwin/globals.cc
+++ b/winsup/cygwin/globals.cc
@@ -74,7 +74,7 @@ bool pipe_byte;
 bool reset_com;
 bool wincmdln = true;
 winsym_t allow_winsymlinks = WSYM_deepcopy;
-bool disable_pcon = true;
+bool disable_pcon;
 bool winjitdebug = false;
 bool nativeinnerlinks = true;
 
